multinet/swagger/workspace_tables.yaml

```diff
@@ -0,0 +1,21 @@
+Retrieve the tables of a single workspace.
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table_type"
+
+responses:
+  200:
+    description: A list of table names belonging to the specified workspace
+    schema:
+      type: array
+      items:
+        $ref: "#/definitions/entity_name"
+
+  404:
+    description: Specified workspace could not be found
+    schema:
+      $ref: "#/definitions/entity_name"
+
```

Comment:
Can return a 400 from BadQueryArgument if type is wrong

Reply:
e68172930be12b3026b1e72593c333e1f5e839c4

Also made this change to the `node_edges` endpoint.